common-arcana.lic

```diff
@@ -24,17 +24,20 @@ module DRCA
 
     loop do
       pause 5 while mana <= amount
-      harness_mana(amount) if harness
+      harness_mana(['amount']) if harness
```

Comment:
Won't this pass 'amount' as an arg, instead of the value of the `amount` variable?

Reply:
Yes, fixed it. Thanks!